src/main/java/ru/skypro/homework/controller/AdController.java

```diff
@@ -17,136 +17,56 @@
 @CrossOrigin(value = "http://localhost:3000")
 @RestController
 @RequiredArgsConstructor
+@RequestMapping("/ads")
 public class AdController {
 
     //private final AdService adService;
 
     //
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "200", description = "запрос выполнен, вывод информации об объявлениях;"
-            )
-    }
-    )
-    @GetMapping("/ads")
+    @GetMapping
     public ResponseEntity<Ads> adInfo() {
             return ResponseEntity.ok().build();
     }
 
     //
 
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "201", description = "запрос выполнен, вывод информации об объявлениях;"
-            ),
-            @ApiResponse(
-                    responseCode = "401", description = "пользователь не авторизирован"
-            )
-    }
-    )
-    @PostMapping("/ads")
-    public ResponseEntity<?> newAd(@RequestBody Ad ad) {
+    @PostMapping
```

Comment:
```suggestion
    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
вот так должно быть, иначе не будет работать